src/components/NewModal/DefaultActionBar.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable import/prefer-default-export */
+import React from "react";
+import CrossIcon from "../Icons/Cross";
+import IconButton from "../Button/IconButton";
+import { colors } from "../../theme/index";
+
+import { CancelButtonContainer } from "./index.styles";
+
+import { ModalConsumer } from "./context";
+
+const DefaultActionBar = () => (
+  <ModalConsumer>
+    {({ closeModal }) => (
```

Comment:
I think we should decouple `DefaultActionBar` from `ModalConsumer`. Rather than having `ModalConsumer` as a child, let's pass an `onClose` prop into `DefaultActionBar` to `IconButton`. This way, we can wrap  `DefaultActionBar` with `ModalConsumer` or pass a different handler.

Reply:
Nice idea, thanks